### DISCUSSION TITLE
Are there any limitations of the browser based version vs a local install?

### DISCUSSION THREAD

**Discussion by RileyEv** [foxglove/community#62] [Q&A] [answered]
Are there difference that I need to bear in mind if using foxglove in the browser? Are there any features that aren't supported or are more restrictive?

I understand that the local install runs with electron, so I guess there isn't a huge difference, but I wan't to double check :smile:

**Comment by jtbandes** [comment, marked as answer]
There are indeed some differences due to the inherent limitations of the browser environment. It's a great question, and one that I wish we had a documentation page to answer directly — we'll work on that!

Features that are desktop-only:
- Native [live connections](https://foxglove.dev/docs/studio/connection/live-connection) to ROS 1, ROS 2, and Velodyne LiDAR – note that ROS 1/2 can both be used in the web app via [Rosbridge](https://foxglove.dev/docs/studio/connection/live-connection#rosbridge)
- [`foxglove://` URL scheme](https://foxglove.dev/docs/studio/building-and-sharing-links) to open the app
- Support for `package://` URLs for loading [URDF/mesh resources](https://foxglove.dev/docs/studio/panels/3d#marker)
- Support for [extensions](https://foxglove.dev/docs/studio/extensions/getting-started)

**Reply by RileyEv** [reply]
Amazing! Thank you 😊

**Reply by jtbandes** [reply]
This info has been added to https://foxglove.dev/docs/studio/getting-started. Thanks!
